akka-typed-tests/src/test/java/jdocs/akka/typed/IntroTest.java

```diff
@@ -39,7 +39,7 @@ public Greeted(String whom) {
     public static final Behavior<Greet> greeter = Actor.immutable((ctx, msg) -> {
       System.out.println("Hello " + msg.whom + "!");
       msg.replyTo.tell(new Greeted(msg.whom));
-      return Actor.same();
+      return Behavior.same();
```

Comment:
I don't think we should use that in examples. Everything you need is in `Actor`, is a better default, but for example when implementing a custom behavior (`ExtensibleBehavior`) it's more natural to use `Behavior.some`.